src/app/core/utils/ComponentUtils.ts

```diff
@@ -158,15 +168,17 @@ export function GetAllPaths(obj: Component): Array<Wire | Node> {
  * @param  objects The list of objects
  * @return         A SeparatedComponentCollection of the objects
```

Comment:
Please add a short description of the `full` parameter to these javadoc comments